frontend/src/pages/chart/chart.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Chart from 'chart.js';
+
+
+function Radar(props) {
+    let coverage = props.coverage;
```

Comment:
*nitpick* : I find let {coverage, prop2, prop3, etc} = props to be more readable and reduce code size

Reply:
Addressed in afb14ddec0451920a81cdf939abbc84f9ba788be